### DISCUSSION TITLE
Any plans to make feed more interactive? more like a place where discussions start?

### DISCUSSION THREAD

**Discussion by derberg** [community/community#38478] [Other Feature Feedback, Questions, & Ideas]
I wrote an article about how I think GitHub could be a better place for discussions than it is now with Twitter -> https://www.brainfart.dev/blog/town-square-for-tech

I also suggested there one nice possible addition:
```
Dedicated feed type for juniors, for first-time contributors to explore not just good-first-issues but also recommend projects that match your interest and your pet projects. 
If you have some projects written in JavaScript, then the feed would suggest similar ones and pick only those that are friendly for first-time contributors.
```

**Comment by david-engelmann** [comment]
It feels like the For You section which is currently in beta is going to take this approach. It will be right next to the feed, check out the screenshot below!

![image](https://user-images.githubusercontent.com/81196676/200386141-63f530de-00d2-499c-8e62-34fc5c627de0.png)

**Reply by derberg** [reply]
Exactly this motivated me to write an article. In the article, I actually share screenshot from my `For you`.

This topic is under `Feed` category, I thought GitHub Feed is exactly this
